modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/settings/SettingsBuilder.java

```diff
@@ -15,7 +16,7 @@
 /**
  * @author André de Oliveira
  */
-public class SettingsBuilder {
```

Comment:
Thats the way I did too, it just weird to have builder implementing helper

Reply:
ya, i just renamed SettingsBuilder to SettingsHelperImpl